xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_zh_Hans.properties

```diff
@@ -0,0 +1,5412 @@
+# ---------------------------------------------------------------------------
```

Comment:
@tmortagne I have a doubt: zh_Hans is supposed to be supported by Java?

Reply:
Doing `new Locale(zh_Hans)` wrong crash but the right code is `zh_CN` (see Locale#SIMPLIFIED_CHINESE) and in the context of XWiki this is actually `zh` since we decided to make it the default variant.

Reply:
And apparently right now we have a user dedicated to do translations for zh_Hans :-/

Reply:
Maybe users can create the language they want and the upgrade restored `zh_HANS`.

Reply:
@sugov5 thanks for your contribution on XWiki translations! As specified on this thread by @tmortagne for technical reasons the actual translation in Chinese (simplified) have to be made with the code `zh` not `zh_Hans`. The language with the code `zh_Hans` was actually supposed to be deleted from our Weblate platform, but apparently it has been created back. So we are going to delete it again, sorry for the inconveniance.